### ISSUE TITLE
Chrome shows homepage in stead of postman application

### ISSUE THREAD

**Issue by BertusVanZyl** [a85/POSTMan-Chrome-Extension#6022]
When running the shortcut after installing, chrome opens to its home page in stead of running the postman app. When I run the installer, after the installer completes, postman is opened successfully. So to run postman, I have to reinstall the app from the installer every time.

**App information (please complete the following information):**
Chrome: Version 72.0.3626.121 (Official Build) (64-bit)
Postman: Postman-win64-6.7.4
Windows: Windows 10 Pro 1803 (OS Build 17134.590)

**Comment by numaanashraf** [comment]
@FrikkinLazer Do you have Postman Chrome App also installed?

**Comment by BertusVanZyl** [comment]
It was installed. I removed it, and reinstalled from Postman-win64-6.7.4-Setup.exe. After installation, postman started up. I closed postman, and when I used the shortcut, it still opened chrome to the chrome homepage. Is there something I can test or check to debug?

**Comment by Raj-postman** [comment]
@BertusVanZyl Apologies for the delay in our response. May I know if you are still facing this issue using the latest version of the application (v.7.24.0)?

**Comment by BertusVanZyl** [comment]
Hi there. I just installed the latest version, and did a quick analysis.

After installing the latest version, this is the situation.

When searching for "Postman" in windows this is the result:
![image](https://user-images.githubusercontent.com/2371314/82432932-026f3780-9a91-11ea-9d2a-e9ea08c7a9fd.png)

The problem is that there are two links to choose from. One works, the other does not, and windows randomly chooses one to be the top search result.

The one that does not work points here: "AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Chrome Apps\Postman.lnk". This link just opens chrome, there are no command line options at all. 

The other link works perfectly.

My guess is that an older version of postman created the non working .lnk file ages ago, and the issue has been resolved. I would not worry too much about this issue. I would close it, and reopen it if someone else gets a similar problem in future. 

I am just going to delete the non working .lnk file so that windows does not show it as a result anymore.
